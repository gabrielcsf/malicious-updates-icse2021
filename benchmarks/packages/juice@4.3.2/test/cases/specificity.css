#wrap #a { /* [2, 0, 0] - winner */
  color: red;
}
#wrap a { /* [1, 0, 1] */
  color: #000;
}
#a { /* [1, 0, 0] */
  color: #00f;
}
div[a=b] .wrap div { /* [0, 2, 2] */
  display: block;
}
#wrap .wrap { /* [1, 1, 0] - winner */
  display: inline-block;
}
#wrap div.wrap #a { /* [2, 1, 1] */
  background: #000;
}
#wrap[a=b] div.wrap #a { /* [2, 2, 1] - winner */
  background: green;
}
#wrap[a=c] div.wrap #a { /* [2, 2, 1] - no match */
  background: #00f;
}

#wrap .wrap #a { /* [2, 1, 0] */
  background: #f00;
}
#wrap .wrap #a { /* [2, 1, 0] - winner */
  border-color: green;
}
div #wrap-2 #a { /* [2, 0, 1] */
  border-color: #008000;
}
