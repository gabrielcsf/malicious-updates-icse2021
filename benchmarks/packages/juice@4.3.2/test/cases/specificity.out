<html><body>
<div id="wrap" a="b">
  <div class="wrap" id="wrap-2" style="display: inline-block;">
    <div id="a" style="display: block; color: red; border-color: green; background: green;"></div>
  </div>
</div>
</body></html>
